patroni/postgresql.py

```diff
@@ -120,16 +138,19 @@ def initialize(self):
     def delete_trigger_file(self):
         os.path.exists(self.trigger_file) and os.unlink(self.trigger_file)
 
-    def sync_from_leader(self, leader):
-        r = parseurl(leader.conn_url)
-
+    def write_pgpass(self, record, append=False):
         pgpass = 'pgpass'
-        with open(pgpass, 'w') as f:
+        with open(pgpass, 'w' if not append else 'a') as f:
```

Comment:
Append is not necessary, as except for pg_rewind this file is only used by pg_basebackup, and not by streaming replication (which gets the pass from recovery.conf).